tests/test_templates.py

```diff
@@ -534,7 +539,19 @@ def make_n_features(self, n):
                   (QAOAEmbedding,
                    {'features': [1., 2.], 'weights': qaoa_embedding_normal(n_layers=2, n_wires=1), 'wires': range(1)}),
                   (QAOAEmbedding,
-                   {'features': [1., 2.], 'weights': qaoa_embedding_uniform(n_layers=2, n_wires=1), 'wires': range(1)})
+                   {'features': [1., 2.], 'weights': qaoa_embedding_uniform(n_layers=2, n_wires=1), 'wires': range(1)}),
+                  (CnotRingLayers,
+                   {'weights': cnot_ring_layers_uniform(n_layers=3, n_wires=1), 'wires': range(1)}),
```

Comment:
Although easy to check, `n_layers=1` would be one of the simplest edge cases. I see that this holds for other templates as well, though I guess if it's included it could be included simply for this template only.

Reply:
I added a test case of ``n_layers=1, n_wires=1``. Was that what you had in mind?

Reply:
Sure, sounds good to me :)